scripts/tests/chiptest/__init__.py

```diff
@@ -160,6 +160,57 @@ def _GetInDevelopmentTests() -> Set[str]:
     }
 
 
+def _GetDarwinFrameworkToolUnsupportedTests() -> Set[str]:
+    """Tests that fail in darwin-framework-tool for some reason"""
+    return {
+        "DL_LockUnlock",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_AddNewFabricFromExistingFabric",  # darwin-framework-tool does not support the GetCommissionerRootCertificate command.
+        "TestAttributesById",  # The name of the arguments once converted differs for chip-tool and darwin-framework-tool
+        "TestBasicInformation",  # darwin-framework-tool does not support writing readonly attributes by name
+        "TestClusterComplexTypes",  # Test Failure : The test expectation "wasPresent (0) == True" is false
+        "TestClusterMultiFabric",  # When reading TestFabricScoped, the result differs
+        "TestCommandsById",  # darwin-framework-tool does not support writing readonly attributes by name
+        "TestDiagnosticLogs",  # darwin-framework-tool does not implement a BDXTransferServerDelegate
+        "TestDiscovery",  # darwin-framework-tool does not support dns-sd commands.
+        "TestEvents",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "TestEventsById",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "TestGroupMessaging",  # darwin-framework-tool does not support group commands.
+        "TestIcdManagementCluster",  # darwin-framework-tool does not support ICD registration
+        "TestUnitTestingClusterMei",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "TestReadNoneSubscribeNone",  # darwin-framework-tool does not supports those commands.
+
+        "Test_TC_ACE_1_6",  # darwin-framework-tool does not support group commands.
+        "Test_TC_ACL_2_5",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_6",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_7",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_8",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_9",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_10",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_BINFO_2_1",  # darwin-framework-tool does not support writing readonly attributes by name
+        "Test_TC_BINFO_2_2",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_BRBINFO_2_1",  # Test Failure : The test expects the "UNSUPPORTED_ATTRIBUTE" error but the "FAILURE" error occured.
```

Comment:
Why?  If we don't understand why, there should be a tracking issue to figure it out, at least, documented here.

Reply:
I have tried to add a comment to explain it.